v21.2/known-limitations.md

```diff
@@ -316,12 +275,6 @@ As a workaround, take a cluster backup instead, as the `system.comments` table i
 
 [Tracking GitHub Issue](https://github.com/cockroachdb/cockroach/issues/44396)
 
-### Slow (or hung) backups and queries due to write intent buildup
```

Comment:
@aliher1911, please confirm that it's ok to remove this limitation from the 21.2 docs. Full text of the limitation is here: _includes/v21.2/known-limitations/write-intent-buildup.md.